pages/glossary/different-hue.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Different hue
+key: different-hue
+unambiguous: true
+objective: true
+---
+
+Two sets of colors have _different hue_ if the hue of any color in one set is different from the hue of any color in the other set.
```

Comment:
What I'm trying to convey is two sets have different hue if I can find one pair composed by a color from set 1 and set 2 where the two colors have a different hue. 
Using "each" instead of "any" my interpretation is that would have to be true for every pair of colors instead of one pair of colors.
@EmmaJP do you want to chime in?

Reply:
I do understand the current text the same way as you 😀 (one pair is different means the set are different).

We can probably make that interpretation more obvious:
```suggestion
Two sets of colors have _different hue_ if the corresponding sets of hues are different.
```
(using set equality in the way)

or
```suggestion
Two sets of colors have _different hue_ if there is at least one color from the first set having a hue different from at least one color in the second set.
```